docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.7"
+
+services:
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8000
+    environment:
+      DATABASE_URL: "${DB_TYPE:-postgres}://${DB_USER:-admin}:${DB_PASSWORD:-password}@${DB_HOST:-db}:${DB_PORT:-5432}/${DB_NAME:-pythondotorg}"
```

Comment:
We already have `DATABASE_URL` environment variable: https://pythondotorg.readthedocs.io/install.html#manual-setup

Reply:
Could you explain how it works because if I don't set the `DATABASE_URL` variable the web container will fail.

Reply:
What I meant is we can just specify `DATABASE_URL: "postgres://admin:password@db:5432/pythondotorg"` instead of introducing `DB_*` env variables.